hasura/migrations/1575910000000_init/up.sql

```diff
@@ -8,13 +8,22 @@ CREATE TYPE account_type AS ENUM (
   'DISCOURSE'
 );
 
-CREATE TYPE player_rank AS ENUM (
-  'PLAYER',
-  'BRONZE',
-  'SILVER',
-  'GOLDEN',
-  'PLATINIUM',
-  'DIAMOND'
+CREATE TYPE enneagram_type AS ENUM (
+  'REFORMER',
+  'HELPER',
+  'ACHIEVER',
+  'INDIVIDUALIST',
+  'INVESTIGATOR',
+  'LOYALIST',
+  'ENTHUSIAST',
+  'CHALLENGER',
+  'PEACEMAKER'
+);
+
+CREATE TYPE guild_type AS ENUM (
```

Comment:
For the guild types, since we will likely be changing this and adding more, using the PG native enum type has some drawbacks, would be good to set it up as foreign-key references: https://hasura.io/docs/1.0/graphql/manual/schema/enums.html#enums-in-a-database